benchmark/comprehensive_benchmark.dart

```diff
@@ -4,9 +4,9 @@ import 'package:benchmark_harness/benchmark_harness.dart';
 import 'package:ordered_set/comparing.dart';
 import 'package:ordered_set/ordered_set.dart';
 
-const _maxOperations = 2500;
+const _maxOperations = 1000;
 const _maxElement = 10000;
-const _startingSetSize = 500;
+const _startingSetSize = 250;
```

Comment:
this is just to put it in line with the time of the other tests. with the new impl it is totally fine to go even above these numbers